cmd/maya-apiserver/spc-watcher/handler.go

```diff
@@ -139,3 +147,40 @@ func (c *Controller) getSpcResource(key string) (*apis.StoragePoolClaim, error)
 	}
 	return spcGot, nil
 }
+
+func syncSpc(spcGot *apis.StoragePoolClaim) error {
+	if len(spcGot.Spec.Disks.DiskList) > 0 {
+		// TODO : reconciliation for manual storagepool provisioning
+		return fmt.Errorf("no reconciliation for manual provisioning for storagepoolclaim %s", spcGot.Name)
+	}
```

Comment:
@avishnu 
Review above code once.
Manual provisioned pool was also getting into reconciliation causing undesired behaviour as its reconciliation
will be somewhat different from automated provisioned pool.
Had added a todo comment in code too.

Reply:
Ok. Please track this as a future item.